### ISSUE TITLE
Aria landmark roles to components we typically place at the root

### ISSUE THREAD

**Issue by knice** [ucsc/truss#27]
From Graham:

> Accessibility fix requested (possibly low-impact, but widespread): The top accessibility issue flagged by SiteImprove on our sites is "Text not included in an ARIA landmark". The element in question is the <trss-ucsc-header> which may not be recognized by screen readers because it lacks an explicit ARIA landmark role. Adding role="banner" to <trss-ucsc-header> should align the element with WCAG best practices.
>
> Let me know if this fix is worthwhile and accurate, otherwise I can designate the issue in SiteImprove as "Can't fix" or "False positive". Thanks!
